### ISSUE TITLE
[Enhancement] When updating, prompt once for both updates in the repositories and the AUR

### ISSUE THREAD

**Issue by gabrc52** [manjaro/pamac#319]
When updating, after it's updated every package in the repositories, Pamac demands attention and prompts again for AUR repositories.

I think it should prompt only once so it updates all (selected) packages at once.

**Comment by gabrc52** [comment]
Oh wait, duplicate of #316
